Earthfile

```diff
@@ -57,4 +58,12 @@ repo-catalyst-voices-all:
     WORKDIR /repo
     COPY --dir catalyst_voices catalyst_voices_packages utilities melos.yaml pubspec.yaml  .
 
-    SAVE ARTIFACT /repo repo
\ No newline at end of file
+    SAVE ARTIFACT /repo repo
+
+# check-cat-voices-license - Check the license of catalyst_voices.
+check-cat-voices-license:
```

Comment:
Consider to move it to /catalyst_voices/Earthfile where we keep cat-voices related checks.

Does this run separately for every dart/flutter package or only for catalyst_voices package? There might be in the future packages that are not imported into catalyst_voices so you might skip validating them.

If it's easy enough to update it so that it runs for every package - lets do it. You can have a a look at melos.yaml which is a configuration for the tool that discovers all packages automatically, i.e. you can call `melos exec -- {your_command}` and the tool will automatically run `{your_command}` for every package that it finds, not only for catalyst_voices

Reply:
This license checker rely on `pubspec.yaml`, so every direct and transitive dependencies mention in pubspec will be checked. I will move this target to `cat-voice`

Reply:
True, but if we have a package in the future that isn't the dependency of catalyst_voices then it won't be checked - that was the reason we use melos, it's able to handle these cases of unrelated packages.

Reply:
I see, I thought we will only check for packages that is used in cat-voice

Reply:
Best would be to do it for all packages :) You can check with Steven if it's too difficult for now, maybe he'll agree to postpone.